src/web_app/js/util.js

```diff
@@ -118,15 +107,27 @@ function parseJSON(json) {
 }
 
 // Filter a list of TURN urls to only contain those with transport=|protocol|.
-function filterTurnUrls(urls, protocol) {
-  for (var i = 0; i < urls.length;) {
-    var parts = urls[i].split('?');
-    if (parts.length > 1 && parts[1] !== ('transport=' + protocol)) {
-      urls.splice(i, 1);
-    } else {
-      ++i;
+function filterTurnUrls(config, protocol) {
```

Comment:
Can we just filter each iceServer in the turnServer response? Not sure why the function had to be rewritten.